_benchmark/reference/commands/command-flags.md

```diff
@@ -290,4 +290,34 @@ The name and path used for the chart's output. Default is `stdout`.
 ## limit
 <!-- vale on -->
 
-Limits the number of search results for recent test runs. Default is `10`.
\ No newline at end of file
+Limits the number of search results for recent test runs. Default is `10`.
+
+<!-- vale off -->
+## latency-percentiles
+<!-- vale on -->
+
+Specifies a list of latency percentiles to report after the workload runs. Default is `50,90,99,99.9,99.99,100`. 
+
+<!-- vale off -->
+## throughput-percentiles
+<!-- vale on -->
+
+Specifies a list of throughput percentiles to report after the workload runs, in addition to mean/median/max/min. Default is `None`. 
```

Comment:
Is this to be specified as floating point numbers?  Please elaborate here.

Reply:
It can be taken as an int, float, or str. Based on [the code here](https://github.com/opensearch-project/opensearch-benchmark/blob/main/osbenchmark/metrics.py#L1333-L1337) that Peter added, it converts the user inputs to floats.